dashboard/app/models/skill.rb

```diff
@@ -8,9 +8,42 @@
 #  concept             :string(255)
 #  created_at          :datetime         not null
 #  updated_at          :datetime         not null
+#  key                 :string(255)      not null
+#
+# Indexes
+#
+#  index_skills_on_key  (key) UNIQUE
 #
 class Skill < ApplicationRecord
   validates :description, presence: true
 
   has_and_belongs_to_many :levels, join_table: 'levels_skills'
+
+  def seeding_key(seed_context)
+    {'skill.key': key}.stringify_keys
+  end
```

Comment:
is this necessary? it seems like you can just use the key directly. usually we only use seeding_key when multiple fields are required to uniquely define the object.